client/src/connection/itc/index.ts

```diff
@@ -13,10 +20,13 @@ import {
 import { updateStatusBarItem } from "../../components/StatusBarItem";
 import { extractOutputHtmlFileName } from "../../components/utils/sasCode";
 import { Session } from "../session";
+import { LineParser } from "./LineParser";
+import { ERROR_END_TAG, ERROR_START_TAG } from "./const";
 import { scriptContent } from "./script";
 import { LineCodes } from "./types";
+import { decodeEntities } from "./util";
 
-const SECRET_STORAGE_NAMESPACE = "ITC_SECRET_STORAGE";
+const SECRET_STORAGE_NAMESPACE = "ITC_SECRET_STORAGE-1";
```

Comment:
is this change necessary?

Reply:
d'oh. Updated in https://github.com/sassoftware/vscode-sas-extension/pull/842/commits/d7f28f15d5c4ec6925932a8f3ac546b28e816ecd